Simplenote/src/main/java/com/automattic/simplenote/models/Preferences.java

```diff
@@ -0,0 +1,54 @@
+package com.automattic.simplenote.models;
+
+import com.simperium.client.BucketObject;
+import com.simperium.client.BucketSchema;
+
+import org.json.JSONException;
+import org.json.JSONObject;
+
+public class Preferences extends BucketObject {
+    public static String PREFERENCES_OBJECT_KEY = "preferences-key";
+
+    private static final String BUCKET_NAME = "preferences";
+    private static final String ANALYTICS_ENABLED_KEY = "analytics_enabled";
+
+    private Preferences(String key, JSONObject properties) {
+        super(key, properties);
+    }
+
+    public boolean getAnalyticsEnabled() {
+        try {
+            return getProperties().getInt(ANALYTICS_ENABLED_KEY) > 0;
```

Comment:
Note: This should be a normal boolean instead of the int > 0 check, but iOS seems to write a boolean to simperium as an integer: Automattic/simplenote-ios#228 (comment)

Depending on if that gets fixed, I'll do a quick follow-up PR to set this to a regular ol' boolean.